src/Exchange.ts

```diff
@@ -547,7 +549,7 @@ export class Exchange {
    * @param payload Body of POST request.
    */
   private _postRequest(path: string, payload: any): Promise<any> {
-    return this.utils.fetchUrl(path, {
+    return this.provider.fetchEndpoint(path, {
```

Comment:
I would make this an own method of the provider to
`postEndpoint(endpoint, payload)`

Does not have to be in this PR but if you agree open an issue.

Reply:
yes will open an issue